servicecontrol/creating-config-file.md

```diff
@@ -76,70 +76,110 @@ net stop particular.servicecontrol
+
+Default: `65512`.
+___
+##### ServiceControl/HoursToKeepMessagesBeforeExpiring (int)
+
+The number of hours to keep a message for before it is deleted, 
 
-* `ServiceControl/HoursToKeepMessagesBeforeExpiring` (int)
+Default: `720` (30 days). Valid Range is `24` (1 day) through to `1440` (60 days)
+___
+##### ServiceControl/DbPath (string)
 
-  The number of hours to keep a message for before it is deleted, Default `720` (30 days). Valid Range is `24` (1 day) through to `1440` (60 days)
+The path where the internal RavenDB is located. 
 
-* `ServiceControl/DbPath` (string)
+Default: `%SystemDrive%\ProgramData\Particular\ServiceControl\`
+___
+##### ServiceControl/TransportType (string)
 
-  The path where the internal RavenDB is located. Default `%SystemDrive%\ProgramData\Particular\ServiceControl\`
+The transport type to run ServiceControl with. 
 
-* `ServiceControl/TransportType` (string .Net type)
+Default: `NServiceBus.Msmq, NServiceBus.Core`
+___
+##### NServiceBus/Transport  (string)
 
-  The transport type to run ServiceControl with. Default `NServiceBus.Msmq, NServiceBus.Core`
+The connection string for the transport. This setting should be placed in `connectionStrings` section of configuratoin file.
+___
+##### ServiceBus/AuditQueue (string)
 
-* `NServiceBus/Transport` (string)
+The audit queue name. 
 
-  The connection string for the transport. This setting should be placed in `connectionStrings` section of configuratoin file.
+Default: `audit`
+___
+##### ServiceBus/ErrorQueue (string)
 
-* `ServiceBus/AuditQueue` (string)
+The error queue name. 
 
-  The audit queue name. Default `audit`
+Default: `error`
+___
+##### ServiceBus/ErrorLogQueue (string)
 
-* `ServiceBus/ErrorQueue` (string)
+The error queue name to use for forwarding error messages. 
 
-  The error queue name. Default `error`
+Default: `<ErrorQueue>.log`
+___
+##### ServiceBus/AuditLogQueue (string)
 
-* `ServiceBus/ErrorLogQueue` (string)
+The audit queue name to use for forwarding audit messages. This only works if `ServiceControl/ForwardAuditMessages` is true. 
 
-  The error queue name to use for forwarding error messages. Default `<ErrorQueue>.log`
+Default: `<AuditQueue>.log`
+___
+##### ServiceControl/MaxBodySizeToStore (int) 
 
-* `ServiceBus/AuditLogQueue` (string)
+Up until version _1.6_ ServiceControl only stores bodies of audit messages that are smaller than 100Kb by default. After version _1.6_ Increase this number to store messages with larger bodies. Messages that have a larger message body in bytes than MaxBodySizeToStore are not stored for audit. This is to ensure that the majority of our users enjoy the best level of performance. For users with special analysis needs, edit MaxBodySizeToStore in ServiceControl.exe.config to increase the size of storeable audit messages.
```

Comment:
we generally dont have any formatting for version numbers in other pages. so no need to make them italic